numba/core/datamodel/models.py

```diff
@@ -179,7 +179,7 @@ def from_argument(self, builder, val):
 @register_default(types.Boolean)
 @register_default(types.BooleanLiteral)
 class BooleanModel(DataModel):
-    _bit_type = ir.IntType(1)
+    be_type = _bit_type = ir.IntType(1)
```

Comment:
```suggestion
    _bit_type = ir.IntType(1)
```

Couldn't find any place where `be_type` as actually used.

Reply:
Oh, it can be used inside one of the functions you're generating here.